src/Random.h

```diff
@@ -233,6 +246,25 @@ class Random : public RefCounted {
 		return o;
 	}
 
+	// interval (-1,1)
+	// triangle distribution at p=1
+	// increasing steepness of normal distribution at p > 1
+	inline fixed SFixed(int p)
+	{
+		fixed o = Fixed();
+		o -= Fixed();
+		while (--p > 0)
+			o *= (fixed(1, 4) + Fixed() * fixed(3, 4));
+		return o;
+	}
+
+	// interval (mean - stddev, mean + stddev)
+	// this is not a true gaussian distribution
+	inline fixed NormFixed(fixed mean, fixed stddev)
```

Comment:
`stddev` was only chosen because the function was attempting to mimic the API of `Normal(mean, stddev)`. I don't mind re-naming it, as the parameter is indeed intended to be a hard limit on the maximum value produced by the function.

That graph for `FixedFixedFixed/3` looks significantly better than the approximation I came up with, thank you! I kludged together the existing SFixed() approximation with the help of an online probability visualization tool that wasn't very good.

Reply:
Out of curiosity, what is the graph of two calls to the new implementation multiplied against each other compared to the graph of SFixed(4) or SFixed(5)? I'm entirely replacing SFixed() with the new NormFixed() implementation, and I'd like to know how much error compared to the previous iteration I'm introducing here.

Reply:
![изображение](https://github.com/pioneerspacesim/pioneer/assets/18342621/382449f2-2b44-418a-8779-a16723f6b48d)

I put all three graphs (scaled down 10 times vertically), you can see that `NormFixed()*NormFixed()` is the sharpest, although the peak is the same as that of `SFixed(5)`.
If maybe you want to play around with it, and you're not afraid of your eyes bleeding, I pushed my [test mess](https://github.com/Gliese852/gles_imgui/tree/histogram), the shape of the graphs is defined [here](https://github.com/Gliese852/gles_imgui/blob/571750eb739760bbec710b6ed21e086301af8ae8/main.cpp#L193).
(In order for the graph to grow, you need to move the mouse, since redrawing occurs only in the event handler)

Reply:
Perhaps useful to subtract the two, and only show the (relative?) differences? (Depending on what is interesting; I'm not really following the details of the discussion)

Reply:
Thanks! That matched what I was able to determine using JupyterLab; I wound up with a slightly different term to control body inclination, though I may still use `NormFixed() * NormFixed()`.